dlp-terraform/ecs/sqs.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_sqs_queue" "training_queue" {
```

Comment:
is there any dead letter queue in place? 

Dead letter queues are "special queues" where you are able to access why a particular queue item didn't get processed due to some error. 

https://aws.amazon.com/what-is/dead-letter-queue/

Reply:
where in the PR?